src/plan_bot.py

```diff
@@ -59,12 +60,18 @@ def build_response_text_pure_plan(plan_record, post):
     )
 
 
-def build_response_text(plan_record, post):
+def build_response_text(plan_record, post, **kwargs):
```

Comment:
My gut is that we should leave this for plans only. (Maybe rename `build_plan_match_response_text` or something). And then keep `build_verbatim_response_text` as a separate method.